tests/model_fields_/test_embedded_model.py

```diff
@@ -123,3 +129,62 @@ class MyModel(models.Model):
         self.assertEqual(
             msg, "Embedded models cannot have relational fields (Target.key is a ForeignKey)."
         )
+
+
+class SubqueryExistsTest(TestCase):
+    def setUp(self):
+        # Create test data
+        address1 = Address.objects.create(city="New York", state="NY", zip_code=10001)
```

Comment:
Embedded models should be saved to the database in their own collection (i.e. no `create()`). #216 will prohibit it.